### ISSUE TITLE
Figure out how to test, ignore, or replace containers

### ISSUE THREAD

**Issue by aeschylus** [ProjectMirador/mirador#2047]
We're currently not testing component containers, partially because there's not much to test. This is slowly affecting overall coverage, so I wonder if we should find a pattern for testing these. One example is writing integration tests that touch this code, but I'm not sure whether codecov will pick them up.

https://twitter.com/dan_abramov/status/704268606742388736?lang=en

**Comment by mejackreed** [comment]
Right now our integration tests aren't picked up in code coverage states. Not sure if there is a way to fix this.

**Comment by charbugs** [comment]
Yes I agree, I think the right place to these containers is in integration tests because the basically only bind modules together.

**Comment by marlo-longley** [comment]
Audit 2026:
 
Confidence is medium because problem is clear
My preferred solution is to migrate off containers? Migrate to a to hooks-based useSelector/useDispatch pattern in components, eliminating the container layer.
This would be a big migration and needs further dev investigation.
